nanopi/duo.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2017 Richard Hull
+# See LICENSE.md for details.
+
+'''
+Created on Dec 31, 2017
+
+@author: sgoldsmith
+
+Usage:
+
+import nanopi.duo
```

Comment:
Use the RST code formatting extensions, so this would become:

```
Usage:

.. code:: python
   import nanopi.duo

   GPIO.setmode(nanopi.duo.BOARD)
```

Reply:
OK, bringing this up to standards :)